testsuite/shared/src/main/java/org/wildfly/test/security/common/elytron/LdapRealm.java

```diff
@@ -0,0 +1,804 @@
+/*
+ * Copyright 2020 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.test.security.common.elytron;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.client.ModelControllerClient;
+import org.jboss.as.controller.operations.common.Util;
+import org.jboss.as.test.integration.management.util.CLIWrapper;
+import org.jboss.as.test.integration.security.common.Utils;
+import org.jboss.dmr.ModelNode;
+
+/**
+ * A {@link ConfigurableElement} to define the ldap-realm resource within the Elytron subsystem.
+ *
+ * @author Ondrej Kotek
+ */
+public class LdapRealm implements SecurityRealm {
```

Comment:
Not directly related to this PR so not a request to change anything.

We end up using these builders in many places, it starts to feel like maybe they should be developed in a module adjacent to the subsystem in WildFly Core so as resources are maintained the test builders can also be maintained.

Reply:
It seems the builders grow enough with new tests, I don't see many maintenance commits: https://github.com/wildfly/wildfly/commits/master/testsuite/shared/src/main/java/org/wildfly/test/security/common/elytron
Have you noticed any issues with this approach?